ohttp/Cargo.toml

```
@@ -32,9 +33,10 @@ chacha20poly1305 = {version = "0.8", optional = true}
 serde = "1.0"
 serde_derive = "1.0"
 toml = "0.5"
+mozbuild = {version = "0.1", optional = true}
 
 [build-dependencies.bindgen]
-version = "0.56"
+version = "0.59"
```

```suggestion
version = "0.61"
```

Though I'm noticing the gecko isn't quite up to that, so I'll keep it at 0.59.